### ISSUE TITLE
OpenOCD init failed 

### ISSUE THREAD

**Issue by HWittlich** [GrumpyOldPizza/ArduinoCore-stm32l0#10]
Dear GrumpyOldPizza,

I tried to upload a Sketch to my B-L072Z-LRWAN1 board, (chose right board and COM port in Arduino).
However, I get the following error message:

```
An error occurred while uploading the sketch
Error: init mode failed (unable to connect to the target)
in procedure 'program' 
in procedure 'init' called at file "embedded:startup.tcl", line 495
in procedure 'ocd_bouncer'
** OpenOCD init failed **
shutdown command invoked
```
Any idea what I could do? Thank you for your help.

**Comment by GrumpyOldPizza** [comment]
Sometimes the STM32L0 chip is in a low power mode, where openocd cannot
connect. Hold down the RESET button while uploading the sketch.

**Comment by HWittlich** [comment]
I am amazed by your in-depth knowledge about all of this... Really incredible.

At first, while holding reset down I got `Error: timed out while waiting for target halted` and Windows 7 told me that openOCD.exe stopped working. Then I tried uploading again, releasing the button shortly after starting the upload. Voila! the code was uploaded and the boards LED turned green. 

Now I just have to figure out why LoRaWAN.joined() returns 0 and doesn't connect to TTN. Perhaps I will have to give it another try outdoor.
